main.js

```diff
@@ -0,0 +1,27 @@
+var titleInput = document.querySelector('#title');
+var bodyInput = document.querySelector('#body');
+var saveButton = document.querySelector('button');
+
+titleInput.addEventListener('input', toggleSaveButton);
+bodyInput.addEventListener('input', toggleSaveButton);
+
+// toggleSaveButton();
+
+function toggleSaveButton() {
+    if (titleInput.value === '' || bodyInput.value === '') {
+        saveButton.classList.add('disabled')
+        saveButton.disabled = true
+    } else {
+        saveButton.classList.remove('disabled')
+        saveButton.disabled = false;
+    }
+}
+toggleSaveButton();
```

Comment:
```suggestion

```
I don't think we need this call since we're already calling the function in our event listeners?